.clang-tidy

```diff
@@ -25,6 +25,11 @@ hicpp-*,
 
 -readability-identifier-naming,
 -readability-use-anyofallof,
+-readability-named-parameter,
+
+-cppcoreguidelines-non-private-member-variables-in-classes,
+
+-hicpp-named-parameter,
```

Comment:
Could you move this up so that they are alphabetically sorted?

Reply:
I was thinking about sorting them by category/prefix as first key.